platform/jvm/JVMPlatformAssembler.kt

```diff
@@ -92,7 +93,10 @@ object JVMPlatformAssembler {
 
         fun extractJDK() {
             createDirectory(Path.of(JDK))
-            shell.execute(command = listOf(JAR, "xf", Path.of("..", options.jdkPath).toString()), baseDir = Path.of(JDK))
+            when (currentOS) {
+                MAC, LINUX -> shell.execute(listOf(TAR, "-xf", options.jdkPath, "-C", JDK))
```

Comment:
During the previous PR review I assumed this was just a piece of technical debt that hadn't been refactored out, since this is the only place we use TAR. But it turns out that we actually strictly need TAR here, because the JDK is packaged as a TAR archive on Mac and Linux, so JAR won't work.